services/core-web/cypress/e2e/majorprojects.cy.ts

```diff
@@ -18,6 +18,12 @@ describe("Major Projects", () => {
   it("should upload and download a document successfully", () => {
     const fileName = "dummy.pdf";
 
+    cy.get('body').then((element) => {
```

Comment:
Depending on the status of the project summary, clicking the button for project description can bring us to the project description tab or the actual project description form page. Just have to include this to make sure we click the `View Project Description Details` button if we do get sent to the Project Description tab.